apps/web/src/config/index.ts

```diff
@@ -27,6 +22,13 @@ export const SENTRY_DSN = window._env_.REACT_APP_SENTRY_DSN || process.env.REACT
 
 export const ENV = window._env_.REACT_APP_ENVIRONMENT || process.env.REACT_APP_ENVIRONMENT;
 
+const blueprintApiUrlByEnv = ENV === 'production' || ENV === 'prod' ? 'https://api.novu.co' : 'https://dev.api.novu.co';
+
+export const BLUEPRINTS_API_URL =
+  window._env_.REACT_APP_BLUEPRINTS_API_URL || isCypress
+    ? window._env_.REACT_APP_BLUEPRINTS_API_URL || process.env.REACT_APP_BLUEPRINTS_API_URL || 'http://localhost:1336'
```

Comment:
`window._env_.REACT_APP_BLUEPRINTS_API_URL` allows to override that URL, should we keep that?

Reply:
we definitely can remove it, I left it is just in case we would want to override it.